### PR TITLE
Add test steps into README

### PR DESCRIPTION
**Pull request by yuxisun1217**
Signed-off-by: Yuxin Sun <yuxisun@redhat.com>

### CODE REVIEW

**Review by teg — COMMENTED** [review]
Thanks for this @yuxisun1217! This looks great! I'll let @Gundersanne have a look before merging, but looks good to me!

File: README.md
Comment (Gundersanne) [inline README.md:8]: I think we can leave the image-builder-frontend and insights-proxy setup out of the image-builder readme.

File: README.md
Comment (Gundersanne) [inline README.md:12]: See above, the proxy is not required to run the integration tests.

File: README.md
Comment (Gundersanne) [inline README.md:65]: This is an internal url, personally don't think it should be in the public readme.

**Review by Gundersanne — CHANGES_REQUESTED** [review]
Thanks! Looks good, some minor change requests.

**Comment by yuxisun1217** [comment]
Thank @Gundersanne ! I removed the image-builder-frontend and insights-proxy repos, removed CI section, and cut some redundant words. Please help to review again.

**Comment by yuxisun1217** [comment]
BTW, what's our custom here if I want to change code in a PR? Force push to the same commit or create a new commit into the same branch? I used to force push before. Thanks!

**Review by Gundersanne — APPROVED** [review]
Nice, thank you

Force pushing is best, alternatively adding commits on top and then using github to squash is also fine (though that only works when there needs to be only 1 commit merged in the end).

**Pull request merged**
